catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -533,6 +546,25 @@ public void init() {
         });
     }
 
+    private Set<String> getHiddenFields(List<Result> metacards) {
+        Set<String> matchedHiddenFields;
+        List<Pattern> hiddenFieldPatterns = configuration.getHiddenAttributes()
+                .stream()
+                .map(Pattern::compile)
+                .collect(Collectors.toList());
+        matchedHiddenFields = metacards.stream()
```

Comment:
prefer to have the variable named to give the return value a little more context. Though I can move the declaration up here where we instantiate.